app/views/documents/_embed_code.html.erb

```diff
@@ -3,11 +3,4 @@
 <% end %>
 <script type="text/javascript">
   DV.load("<%= options[:resource_js_url] %>", <%= data.to_json %>);
-</script>
-<% if data[:pdf] %>
-<noscript>
-  <a href="<%#= options[:pdf_url] %>"><%#= options[:title] %> (PDF)</a>
-  <br>
-  <a href="<%#= options[:full_text_url] %>"><%#= options[:title] %> (Text)</a>
-</noscript>
-<% end %>
```

Comment:
Because we never implemented a generic way of passing resource-type-specific attributes (or the resource itself) into the `options` hash. Follow the `options` assignment backwards and we get to `ApiController.oembed` and then `DC::Embed.embed_for()`.

In this context, `data[:pdf]` was never being triggered anyway (and if it had, the options inside would all be blank), so I just remove this until we resolve the above.